cli/lib/command/internal_command_rebase.dart

```diff
@@ -70,17 +73,27 @@ class InternalCommandRebase extends InternalCommand {
       return;
     }
 
-    final rebaseOnto = targetNode.line.branchNameOrCommitHash();
+    final rebaseData = RebaseData(
+      hasTheirsFlag,
+      hasOursFlag,
+      targetNode.line.branchNameOrCommitHash(),
+      current.localBranchNamesInOrderForRebase().toList(),
+      0,
+    );
+
+    final rebaseFile = File('.git/info/stax/rebase.json');
```

Comment:
You forgot the part `context.getRepositoryRoot()` otherwise if people will call this command from subdirectories json file will be saved in not desired location.